mappings/net/minecraft/client/render/entity/model/BannerBlockEntityModel.mapping

```diff
@@ -1 +1,7 @@
 CLASS net/minecraft/class_550 net/minecraft/client/render/entity/model/BannerBlockEntityModel
+	FIELD field_3309 banner Lnet/minecraft/class_630;
+	FIELD field_3310 stickHorizontal Lnet/minecraft/class_630;
+	FIELD field_3311 stickVertical Lnet/minecraft/class_630;
```

Comment:
Yarn names should be in the usual word order: `horizontalStick`, `verticalStick`.

Reply:
https://github.com/FabricMC/yarn/pull/825/commits/c63132b3b88c2d9d95b23fae485b061d82b69c44